docs/source/concepts/bento.rst

```diff
@@ -728,6 +728,24 @@ Python Options Table
 | wheels            | List of paths to wheels to include in the bento                                    |
 +-------------------+------------------------------------------------------------------------------------+
 
+Models
+^^^^^^
+
+You can specify the model to be used for building a bento using a string model tag or a dictionary, which will be written to the ``bento.yaml`` file in the bento package.
+
```

Comment:
Should also mention that these model configs are used to fetch models from Yatai or BentoCloud, one can download them to local model store by running `bentoml models pull`. Before doing that, one needs to login to Yatai first by running `bentoml yatai login`.

Reply:
Added. Thanks.